test/functional/ui/screen_basic_spec.lua

```diff
@@ -73,33 +73,29 @@ describe('Screen', function()
   describe(':suspend', function()
     it('is forwarded to the UI', function()
       local function check()
-        if not screen.suspended then
-          return 'Screen was not suspended'
-        end
+        eq(true, screen.suspended)
       end
```

Comment:
> Also, how would test failures possibly stop the test execution if not by raising lua error?

I don't know. How does busted decide what a is "test failure" vs an error, in its summary?